### ISSUE TITLE
PCB availability?

### ISSUE THREAD

**Issue by PRKid1** [sensai7/CCVDrum#2]
Hello!

I was wondering if you are still selling the PCA (assembled) for this project.  I just got a Volca Drum and ran into your post on Reddit.  Thank you!

**Comment by gr34tmedia** [comment]
Hi there,  i happen to have the components for the controller and im willingto build you one. You still want one ? Get in contact: lp [at] gr34t.de cheers.

**Comment by PRKid1** [comment]
Yes, I am still interested.  I went to that link, yet, not clear where to place request.  Will try again. UPDATE: sent you an email.
